src/plugins/plot/MctTicks.vue

```diff
@@ -145,7 +151,15 @@ export default {
                 throw new Error('config is missing');
             }
 
-            return config[this.axisType];
+            if (this.axisType === 'yAxis') {
+                if (!this.axisId || this.axisId === config.yAxis.id) {
+                    return config.yAxis;
+                } else {
+                    return config.additionalYAxes.find(axis => axis.id === this.axisId);
+                }
+            } else {
+                return config[this.axisType];
+            }
```

Comment:
Nitpick, but I think this becomes easier to read re-structured as such:
```suggestion
            if (this.axisType !== 'yAxis') {
                return config[this.axisType];
            }
            if (this.axisId && this.axisId !== config.yAxis.id) {
                return config.additionalYAxes.find(axis => axis.id === this.axisId);
            } else {
                return config.yAxis;
            }
```

Reply:
Agreed. Done.